{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/data/loader.py

```diff
@@ -0,0 +1,89 @@
+"""Loads, standardizes and validates input data for the simulation."""
```

Comment:
Add some more to the docstring here as well.  This is our abstraction around the extract and transform pieces of the artifact ETL.  (We might consider renaming the modules :-) ).  No logging is done here because the logging is done in vivarium inputs itself and forwarded.  Give some examples of why this abstraction is important, since the existing code looks like a not very useful wrapper around vivarium inputs interface (maybe link out to bep or other repositories where we've had to use special data not directly accesible from vivairum inputs).

Reply:
Done.